server/server.js

```diff
@@ -15,8 +15,6 @@ var boot = require('loopback-boot');
 var cookieParser = require('cookie-parser');
 var errorHandler = require('strong-error-handler');
 
-app.set('legacyExplorer', false);
-
```

Comment:
This line is not needed because it's already set via `server/config.json`, it's ok to remove it.